Configure step of ``nemsio`` fails with this error:
```
Scanning dependencies of target nemsio
[ 39%] Creating directories for 'nemsio'
[ 40%] Performing download step (git clone) for 'nemsio'
Cloning into 'nemsio'...
Already on 'develop'
Your branch is up to date with 'origin/develop'.
[ 41%] No patch step for 'nemsio'
[ 42%] Performing update step for 'nemsio'
Current branch develop is up to date.
[ 43%] Performing configure step for 'nemsio'
-- The Fortran compiler identification is GNU 9.3.0
-- Check for working Fortran compiler: /usr/bin/f95
-- Check for working Fortran compiler: /usr/bin/f95  -- works
-- Detecting Fortran compiler ABI info
-- Detecting Fortran compiler ABI info - done
-- Checking whether /usr/bin/f95 supports Fortran 90
-- Checking whether /usr/bin/f95 supports Fortran 90 -- yes
-- Setting build type to 'Release' as none was specified.
-- Found MPI_Fortran: /home/dusan/sufs/simple-ufs/libs/mpilibs/local/mpich3/lib/libmpifort.a (found version "3.1") 
-- Found MPI: TRUE (found version "3.1") found components: Fortran 
-- Found bacio: /home/dusan/sufs/NCEPLIBS/build/local/bacio-develop/lib/libbacio_4.a (found version "2.1.0")
CMake Error at CMakeLists.txt:30 (find_package):
  By not providing "Findw3nco.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "w3nco", but
  CMake did not find one.

  Could not find a package configuration file provided by "w3nco" with any of
  the following names:

    w3ncoConfig.cmake
    w3nco-config.cmake

  Add the installation prefix of "w3nco" to CMAKE_PREFIX_PATH or set
  "w3nco_DIR" to a directory containing one of the above files.  If "w3nco"
  provides a separate development package or SDK, be sure it has been
  installed.


-- Configuring incomplete, errors occurred!
See also "/home/dusan/sufs/NCEPLIBS/build/nemsio/src/nemsio-build/CMakeFiles/CMakeOutput.log".
make[2]: *** [CMakeFiles/nemsio.dir/build.make:109: nemsio/src/nemsio-stamp/nemsio-configure] Error 1
make[1]: *** [CMakeFiles/Makefile2:466: CMakeFiles/nemsio.dir/all] Error 2
make: *** [Makefile:84: all] Error 2
```

``nemsio`` depends on ``w3nco``.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -42,6 +42,7 @@ else()
   set(LIBCOMPS_FILE ${LIBCOMPS})
 endif()
 message(STATUS "Installing NCEPLIBS from ${LIBCOMPS_FILE}")
+message(STATUS "Installing NCEPLIBS at ${CMAKE_INSTALL_PREFIX}")
 
 # Enable CRTM if NCEPpost library is desired.
 if(BUILD_POST)
@@ -115,9 +116,8 @@ endforeach()
 # Create CMAKE_PREFIX_PATH list with an alternate separator
 string(REPLACE ";" "|" CMAKE_PREFIX_PATH_ALT_SEP "${_cpath}")
 
-set(nemsio_depends bacio sigio)
+set(nemsio_depends bacio w3nco)
 set(nemsiogfs_depends nemsio)
-set(landsfcutil_depends nemsio)
 set(w3emc_depends nemsio sigio)
 
 foreach(lib ${nceplibs})

--- a/deploy_modulefiles.sh.in
+++ b/deploy_modulefiles.sh.in
@@ -13,7 +13,7 @@ LIBCOMPS=@LIBCOMPS_FILE@
 DFLAT="@FLAT@"
 
 [[ $DOVERWRITEMODULES == "ON" ]] && OVERWRITE="Y" || OVERWRITE="N"
-[[ $DFLAT == "ON" ]] && FLAT ="Y" || FLAT="N"
+[[ $DFLAT == "ON" ]] && FLAT="Y" || FLAT="N"
 
 echo "Begin deploying modules ..."
 

